grammar.js

```diff
@@ -3144,6 +3144,7 @@ module.exports = grammar({
       optional(/[^"]+/),
       '",',
       optional(/[0-9]+/),
+      optional(seq(',', optional(/[0-9]+/))),
```

Comment:
```suggestion
      seq(',', optional(/[0-9]+/)),
```
As far as I can tell
```crystal
#<loc:"",,>
```
is valid, but 
```crystal
#<loc:"",>
```
is a parse error.